test/integration/addresses.js

```diff
@@ -49,6 +47,56 @@ describe('bitcoinjs-lib (addresses)', function () {
     assert.strictEqual(address, '36NUkt6FWUi3LAWBqWRdDmdTWbt91Yvfu7')
   })
 
+  it('can generate a SegWit address (via P2SH)', function () {
+    var keyPair = bitcoin.ECPair.fromWIF('Kxr9tQED9H44gCmp6HAdmemAzU3n84H3dGkuWTKvE23JgHMW8gct')
+    var pubKey = keyPair.getPublicKeyBuffer()
+
+    var redeemScript = bitcoin.script.witnessPubKeyHash.output.encode(bitcoin.crypto.hash160(pubKey))
+    var scriptPubKey = bitcoin.script.scriptHash.output.encode(bitcoin.crypto.hash160(redeemScript))
+    var address = bitcoin.address.fromOutputScript(scriptPubKey)
+
+    assert.strictEqual(address, '34AgLJhwXrvmkZS1o5TrcdeevMt22Nar53')
+  })
+
+  it('can generate a SegWit address', function () {
+    var keyPair = bitcoin.ECPair.fromWIF('Kxr9tQED9H44gCmp6HAdmemAzU3n84H3dGkuWTKvE23JgHMW8gct')
+    var pubKey = keyPair.getPublicKeyBuffer()
+
+    var scriptPubKey = bitcoin.script.witnessPubKeyHash.output.encode(bitcoin.crypto.hash160(pubKey))
+    var address = bitcoin.address.fromOutputScript(scriptPubKey)
+
+    assert.strictEqual(address, 'bc1qt97wqg464zrhnx23upykca5annqvwkwujjglky')
+  })
+
+  it('can generate a SegWit 2-of-2 multisig address (via P2SH)', function () {
+    var pubKeys = [
+      '026477115981fe981a6918a6297d9803c4dc04f328f22041bedff886bbc2962e01',
+      '02c96db2302d19b43d4c69368babace7854cc84eb9e061cde51cfa77ca4a22b8b9'
+    ].map(function (hex) { return Buffer.from(hex, 'hex') })
+
+    var redeemScript = bitcoin.script.multisig.output.encode(2, pubKeys) // 2 of 2
```

Comment:
This redeemScript was fixed in the other PR but not this one.

Perhaps you rebased on the wrong branch?